cmd/repo_set_default_test.go

```diff
@@ -55,12 +58,15 @@ func TestRepoSetDefaultLogs(t *testing.T) {
 				t.Errorf("Did not find expected error '%s' in output", tt.expectedLogs)
 			}
 
-			// check default repo is unchanged and is still incubator
-			output, err = cmdtest.RunAppsody(sandbox, "repo", "list")
-			if err != nil {
-				t.Fatal(err)
+			// If the configDir isn't "" then it is the bad_format_repo which fails to parse when executing repo list
+			if tt.configDir == "" {
```

Comment:
So you just skip the bad_format_repo test? These are all failing tests, so I think the parse error is okay, the test is checking for it and passes when it gets the parse error.

Reply:
This was related to the other comment - when executing `repo list` with the badly formatted config file the test fails. It was just another solution I was trying out whilst trying to debug the issue.
For context, this test fails with this return code when `bad_format_repository_config` is checked at the end too:
```
    --- FAIL: TestRepoSetDefaultLogs/Badly_formatted_repo_config (1.14s)
        /Users/lukeweston/go/src/github.com/appsody/appsody/cmd/repo_set_default_test.go:66: Failed to parse repository file yaml: line 5: mapping values are not allowed in this context
```

Reply:
Seems like you are struggling to get the bad_format_repository tests to work. We may have to rethink them, but I think we should be able to get these tests working as they were before